### ISSUE TITLE
Error in reading URL after sending request

### ISSUE THREAD

**Issue by karstenpatzer** [jmizzoni/mtg-price-fetcher#1]
TLDR: enter http://127.0.0.1:5000/cards?name=<Island> (test), 
Expected result: jsonify list of the details of the top result
Actual result: receive 500 error.
Debug is attached.

[debugg.txt](https://github.com/jmizzoni/mtg-price-fetcher/files/8365939/debugg.txt)

What's going wrong here?

specifically: ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)

^ is the first exception.

Second exception is below:

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/admin/myproject/venv/lib/python3.10/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/admin/myproject/venv/lib/python3.10/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/admin/myproject/venv/lib/python3.10/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/admin/myproject/venv/lib/python3.10/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/admin/myproject/mtg-price-fetcher-master/src/app.py", line 12, in fetch_card_price
    result = get_card_price(cardname, setname)
  File "/Users/admin/myproject/mtg-price-fetcher-master/src/scraper.py", line 83, in get_card_price
    return scrape_price(card_url)
  File "/Users/admin/myproject/mtg-price-fetcher-master/src/scraper.py", line 57, in scrape_price
    card_page.createDom()
  File "/Users/admin/myproject/venv/lib/python3.10/site-packages/htmldom/htmldom.py", line 333, in createDom
    raise Exception
Exception
